pyouroboros/helpers.py

```diff
@@ -20,7 +20,7 @@ def set_properties(old, new, self_name=None):
         'entrypoint': old.attrs['Config']['Entrypoint'],
         'environment': old.attrs['Config']['Env'],
         # networks are conigured later
```

Comment:
There is a typo (from my commit), if you like you can fix it :stuck_out_tongue_winking_eye:

Reply:
I actually removed networking_config none as its being addressed regardless later.

Reply:
:+1: